common/app/navigation/Navigation.scala

```diff
@@ -0,0 +1,157 @@
+package navigation
+
+import common.{Edition, Navigation, editions}
+import _root_.model.Page
+import NavLinks._
+
+import scala.annotation.tailrec
+
+case class Subnav(parent: Option[NavLink], children: Option[Seq[NavLink]], hasSecondaryParent: Boolean = false, hasTertiary: Boolean = false)
```

Comment:
I am not sure I understand the purpose of `hasSecondaryParent` and `hasTertiary`? Can you clarify?